sys/reaper/reaper_unix.go

```diff
@@ -107,3 +118,37 @@ func (m *Monitor) Unsubscribe(c chan runc.Exit) {
 	close(c)
 	m.Unlock()
 }
+
+func notify(e runc.Exit, subscribers []chan runc.Exit) chan struct{} {
+	const timeout = 10 * time.Millisecond
+	var (
+		done  = make(chan struct{}, 1)
+		timer = time.NewTimer(timeout)
+	)
+	timer.Stop()
```

Comment:
I think we need to check timer.Stop to see (even though it's highly unlikely) if we need to drain timer.C first.

Reply:
Oh good catch. I didn't think about this because in my mind when I asked for the change it was a "new timer" per loop. But since its being reused you are 100% correct